docs/src/pages/components/badge.mdx

```diff
@@ -1,2 +1,231 @@
+import { Link } from 'gatsby';
+import { BadgeWithParentExample } from '../../components/BadgeWithParentExample';
+
 # Badge
 
+The Badge component displays a dot or a circle with a count. It is normally used to indicate pending items (items in cart, unread messages, etc.)
+
+## Usage
+
+There are three sizes: `dot` (default) and `small` are meant to be used with an icon (Anchor comes with <Link to="/components/icon">its own set of icons</Link>). `large` is meant to be used standalone.
+
+### Size dot with an icon
+```tsx live
+<div style={{ padding: '2rem 1rem' }}>
+    <Badge
+        borderColor={'white'}
+        borderColorHover={'white'}
+        count={1}
+        offsetBottom={1.2}
+        offsetLeft={0.9}
+    >
+        <Cart scale="lg" />
+    </Badge>
+</div>
+```
+
+### Size small with an icon
+
+To see how `showZero` works, try setting `count` to `{0}`. The badge should disappear. If you set `showZero` to `{true}`, however, the badge should display a `0`.
+
+
+```tsx live
+<div style={{ padding: '2rem 1rem' }}>
+    <Badge
+        borderColor={'white'}
+        borderColorHover={'white'}
+        count={1}
+        offsetBottom={1.1}
+        offsetLeft={0.6}
+        showZero={false}
+        size="small"
+    >
+        <Cart scale="lg" />
+    </Badge>
+</div>
+```
+
+### Size large standalone
+
+This is also an example of the `overflowCount` in action. Try decreasing `overflowCount` or updating `count` to be less than `overflowCount`.
+
+```tsx live
+<div style={{ padding: '2rem 1rem' }}>
+    <Badge
+        borderColor={'white'}
+        borderColorHover={'white'}
+        count={15}
+        overflowCount={9}
+        size="large"
+        standalone={true}
+    ></Badge>
+</div>
+```
+
+### Handling hover states
+
+If a Badge is placed within a component that changes color on hover, Badge needs to update its border color as well. The way this is currently handled is via the `isParentHovered` prop. Badge expects its parent component to communicate hover states by updating that prop.
+
+<BadgeWithParentExample />
+
+Here is an example of Badge with a parent component that uses the `isParentHovered` prop:
+
+```jsx hideTitle
```

Comment:
Check out my documentation on `Card`. I broke up my own example components and then fed them into the `CodePreview` component so that it would allow the user to still play with the overall `Card` component with the components I created. This is not super-straightforward so I can stop by IRL and show you what I mean.

Reply:
Also, if you hide the title it's a good idea to put a new title in it's place. When doing so, it's an h6 tag, so `###### Example` in markdown. Again, check out `Card`'s docs for an example.